### PR TITLE
Combine all images to one

### PR DESCRIPTION
**Pull request by ChrisBr**
foobar

@digitaltom Note that this is untested as I don't have my Cam / Raspi with me...

### CODE REVIEW

**Comment by digitaltom** [comment]
Cool, I can try it out at home later

**Comment by ChrisBr** [comment]
booth1

booth2

@digitaltom also implemented a gallery and link to it!

File: app/assets/templates/picture_set_gallery.html.haml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.container
+  .row
+    .col-lg-3.col-md-4.col-xs-6.thumb{ 'ng-repeat' => 'picture in picture_set.pictures'}
+      %a.thumbnail{ href: "{{picture_set.path}}/{{picture.full}}" }
+        %img.img-responsive{ style: 'width:400px;', src: "{{picture_set.path}}/{{picture.full}}", alt: "" }
+    .col-lg-3.col-md-4.col-xs-6.thumb
+      %a.thumbnail{ href: "{{picture_set.path}}/{{picture_set.combined}}" }
+        %img.img-responsive{ style: 'width:400px;', src: "{{picture_set.path}}/{{picture_set.combined}}", alt: "" }
```
Comment (digitaltom) [inline app/assets/templates/picture_set_gallery.html.haml]: needs to be `ng-src` and `ng-href` when angular variables are used

File: app/models/picture_set.rb [hunk 2]
```diff
@@ -38,6 +40,10 @@ def create
       until !jobs.any?(&:status) do end
       # Merge all polaroid previews to an animated gif
       Syscall.execute("time convert -delay 60 #{date}_*#{POLAROID_SUFFIX} #{date}#{ANIMATION_SUFFIX}", dir: dir)
+      # Merge all images in one combined image
+      Syscall.execute("montage -geometry '25%x25%+25+25<' -background 'LightCoral' " \
+      "-title '#{OPTS.image_caption}' -font '#{OPTS.image_font}' -fill 'white' -pointsize 72 -gravity 'Center' " \
+      "#{date}_*.jpg #{date}#{COMBINED_SUFFIX}")
```
Comment (digitaltom) [inline app/models/picture_set.rb]: I suggest to call it like this: 
```
      # Merge all images in one combined image
      Thread.new do
        Syscall.execute("time montage -geometry '25%x25%+25+25<' -background 'LightCoral' " \
          "-title '#{OPTS.image_caption}' -font '#{OPTS.image_font}' -fill 'white' -pointsize 72 -gravity 'Center' " \
          "#{date}_*.jpg #{date}#{COMBINED_SUFFIX}", dir: dir)
      end
``` 
to offload the work to a background thread (montage takes 14s on my raspi). 
Added correct working dir.

File: app/assets/templates/picture_set_gallery.html.haml (hunk 1 above)
Comment (ChrisBr) [inline app/assets/templates/picture_set_gallery.html.haml]: Huh, really? Just c&p from some other view ...
Will change it!

File: app/models/picture_set.rb (hunk 2 above)
Comment (ChrisBr) [inline app/models/picture_set.rb]: Yeah, it already took some time on my Laptop! Will update it...

**Comment by ChrisBr** [comment]
@digitaltom better late than never ... Updated your suggestions :smile:

**Comment by ChrisBr** [comment]
(However, still untested ... But I can try it out on the weekend)

**Comment by digitaltom** [comment]
@ChrisBr could you fix the rubocop issue?

**Pull request merged**